src/ddo/types.ts

```diff
@@ -68,6 +68,21 @@ export interface Workflow {
   stages: Stage[]
 }
 
+export interface PaymentAttributes {
+  paymentType: 'serviceAgreements' | 'stripe'
+  paymentEnabled: boolean
```

Comment:
what does `paymentEnabled` do?

Reply:
allows metadata like this:
```json
{
  title: 'xxx',
  paymentAttributes: [{
  paymentType: 'stripe'
  paymentEnabled: true,
  additionalAttributes: [{
    stripeAccount: 'xxx-1234'
    }]
  }, {
  paymentType: 'serviceAgreements'
  paymentEnabled: false
  }]
}
```

It could be a simple `enum` but I think this way gives more flexibility allowing to record metadata and enable/disable payment methods in the app